client/client_test.go

```diff
@@ -142,3 +151,33 @@ func TestOpenGetExtendedReportClose(t *testing.T) {
 		}
 	}
 }
+
+func TestGetDerivedKey(t *testing.T) {
+	devMu.Do(initDevice)
+	d := device
+	if err := d.Open("/dev/sev-guest"); err != nil {
+		t.Error(err)
+	}
+	defer d.Close()
+	key1, err := GetDerivedKeyAcknowledgingItsLimitations(device, &SnpDerivedKeyReq{
+		UseVCEK: true,
+	})
+	key2, err := GetDerivedKeyAcknowledgingItsLimitations(device, &SnpDerivedKeyReq{
+		UseVCEK: true,
+		GuestFieldSelect: GuestFieldSelect{
+			GuestPolicy: true,
+		},
+	})
+	key3, err := GetDerivedKeyAcknowledgingItsLimitations(device, &SnpDerivedKeyReq{
+		UseVCEK: true,
+	})
+	if err != nil {
```

Comment:
Shouldn't we check `err` after each call to `GetDerivedKeyAcknowledgingItsLimitations()`?

Reply:
Yes, will fix.